tlf/handle_extension.go

```diff
@@ -123,16 +123,28 @@ type HandleExtension struct {
 
 // String implements the fmt.Stringer interface for HandleExtension.
 // Ex: "(conflicted copy 2016-05-09 #2)"
-func (e HandleExtension) String() string {
+func (e HandleExtension) string(isBackedByTeam bool) string {
 	date := time.Unix(e.Date, 0).UTC().Format(handleExtensionDateFormat)
 	var num string
-	if e.Number > 1 {
+	minNumberSuffixToShow := uint16(2)
+	if isBackedByTeam {
+		// When a TLF is backed by an implicit team, it should always
+		// use the "#1" suffix, unlike for older TLFs.
+		minNumberSuffixToShow = 1
```

Comment:
Is this the number that gets increased when TLF resets?

Reply:
Yeah, within the same day.